### DISCUSSION TITLE
Mobile Version

### DISCUSSION THREAD

**Discussion by FeaturedString** [adipiciu/Travian-scripts#59] [General]
Hi,
So i wonder will there ever by a way to launch your scripts on mobile like iphone or android?

**Comment by adipiciu** [comment]
Yes, it's in the script description.
For example on Android use Firefox with Tampermonkey.

**Comment by adipiciu** [comment]
The scripts work only with the old interface. The new mobile version is totally different and I don't have any plans to support it.
So, make sure you don't check the mobile version option when you log in from your smartphone.
